src/api/config/routes/webui_routes.rb

```diff
@@ -340,6 +340,12 @@
         resources :workflow_runs, only: [:index, :show], controller: 'webui/workflow_runs'
       end
       resources :token_triggers, only: [:show, :update], controller: 'webui/users/token_triggers'
+
+      resource :watched_items, controller: 'webui/watched_items', only: [:toggle], constraints: cons do
+        put '/package/:project/:package/toggle' => :toggle, as: :toggle_package
```

Comment:
This isn't [resourceful routing](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)...

Reply:
We have a clear idea of how to do this but don't want to block this PR. We'll add the changes in a follow-up PR.